docs/examples/general/data_loading/external_input.ipynb

```diff
@@ -95,7 +95,7 @@
    "outputs": [],
```

Comment:
<p>I'm not sure what happened here, but I always thought it should be <code>&lt;class 'cupy.ndarray'&gt;</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/1997/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
Good catch, I'm calling next on wrong iterator.

Reply:
Done